compiler/rustc_trait_selection/src/traits/select/candidate_assembly.rs

```diff
@@ -931,7 +931,10 @@ impl<'cx, 'tcx> SelectionContext<'cx, 'tcx> {
     ) -> Option<ty::PolyExistentialTraitRef<'tcx>> {
         // Don't drop any candidates in intercrate mode, as it's incomplete.
         // (Not that it matters, since `Unsize` is not a stable trait.)
-        if self.infcx.intercrate {
+        //
+        // FIXME(@lcnr): This should probably only trigger during analysis,
```

Comment:
We shouldn't drop any of these candidates frankly... :///

Reply:
It's only necessary bc I think we prefer upcasting candidates, which would trigger a feature error on stable.

Reply:
is there anything you want me to change here :thinking: I don't know what to replace this with

Reply:
No, I guess it can stay as a FIXME